include/utils.php

```diff
@@ -3292,6 +3292,22 @@ function set_focus() {}
     }
 }
 
+/**
+ * Like exit() but will throw an exception if called during tests.
+ *
+ * This is to avoid exit() stopping the test suite without us noticing.
+ *
+ * @param int $status
+ * @throws Exception
+ */
+function sugar_exit($status = 0)
```

Comment:
Actually, we should probably drop the sugar here since it's a new function.

```suggestion
function suite_exit($status = 0)
```